src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp1.1'">$(DefineConstants);netcoreapp11</DefineConstants>
   </PropertyGroup>
```

Comment:
Instead of defining constant, can you add separate file with partial class that's included only for netcoreapp1.1, that way it's more readable, given the number of additions is more.